File: data/src/main/java/com/icehousecorp/maunorafiq/data/city/repository/datasource/CityDataStore.java
package com.icehousecorp.maunorafiq.data.city.repository.datasource;

import com.icehousecorp.maunorafiq.data.city.entity.CityEntity;

import java.util.List;

import rx.Observable;

/**
 * Created by maunorafiq on 12/1/16.
 */

public interface CityDataStore {

    List<CityEntity> getListCityEntity();

    boolean putCityEntity(String city);
}

File: data/src/main/java/com/icehousecorp/maunorafiq/data/city/repository/datasource/CityDataStore.kt
package com.icehousecorp.maunorafiq.data.city.repository.datasource

import com.icehousecorp.maunorafiq.data.city.entity.CityEntity

import rx.Observable

/**
 * Created by maunorafiq on 12/1/16.
 */

interface CityDataStore {

    val listCityEntity: List<CityEntity>

    fun putCityEntity(city: String): Boolean
}
